eva_submission/eload_ingestion.py

```diff
@@ -82,9 +76,20 @@ def ingest(
             self.refresh_study_browser()
 
         if do_variant_load:
+            skip_annotation = False
+            vep_version, vep_cache_version = get_vep_and_vep_cache_version(
+                self.mongo_uri,
+                self.eload_cfg.query(self.config_section, 'database', 'db_name'),
```

Comment:
This `db_name` is set per assembly. Similarly to other section we'll need to check the vep/vep cache versions per analysis.